### ISSUE TITLE
Add more helper methods for session pool

### ISSUE THREAD

**Issue by haoxins** [vesoft-inc/nebula-go#311]
I want to add some helper methods for some common scenarios.

The method and struct should be:

```go
type Tag struct {
  Field   string `nebula:"Field"`
  Type    string `nebula:"Type"`
  Null    string `nebula:"Null"`
  Default string `nebula:"Default"`
  Comment string `nebula:"Comment"`
}

type Edge struct {
  Field   string `nebula:"Field"`
  Type    string `nebula:"Type"`
  Null    string `nebula:"Null"`
  Default string `nebula:"Default"`
  Comment string `nebula:"Comment"`
}
```

```go
type TagSchema struct {
	Name   string
	Fields []TagField
}

type TagField struct {
	Field string
	Type  string
	Null  bool
}

type EdgeSchema struct {
	Name   string
	Fields []EdgeField
}

type EdgeField struct {
	Field string
	Type  string
	Null  bool
}
```

```go
func (pool *SessionPool) ShowTagNames() ([]string, error) {
  ...
}

func (pool *SessionPool) CreateTag(tag TagSchema) error {
  ...
}

func (pool *SessionPool) DescTag(tagName string) ([]Tag, error) {
  ...
}

func (pool *SessionPool) ShowEdgeNames() ([]string, error) {
  ...
}

func (pool *SessionPool) CreateEdge(edge EdgeSchema) error {
  ...
}

func (pool *SessionPool) DescEdge(edgeName string) ([]schema.Edge, error) {
  ...
}
```

cc @Nicole00 @wey-gu

**Comment by Nicole00** [comment]
great idea!
Look forward to your pr!
ps: maybe the EdgeSchema and TagSchema can be the same. and TagField and EdgeField can be the same.

**Comment by haoxins** [comment]
> ps: maybe the EdgeSchema and TagSchema can be the same. and TagField and EdgeField can be the same.

YES!!!
but i can't find a good name for these, any good idea?

**Comment by Nicole00** [comment]
How about LabelSchema and LabelField?  the tag and the edge can be represented by label.

**Comment by haoxins** [comment]
@Nicole00 https://github.com/vesoft-inc/nebula-go/pull/312

**Comment by haoxins** [comment]
closed by #312

**Comment by wey-gu** [comment]
We need this in all SDKs Orz....🫡
